cypress/utils/dashboards/workspace-plugin/commands.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BASE_PATH } from '../../base_constants';
+import { WORKSPACE_API_PREFIX } from './constants';
+
+Cypress.Commands.add('createWorkspace', (workspaceName) => {
```

Comment:
maybe add one more parameter to support workspace features?

Reply:
Maybe an object for better backward capability?

Reply:
Good idea! I will add that